R/pkgdown-package.R

```diff
@@ -2,9 +2,9 @@
 "_PACKAGE"
 
 ## usethis namespace: start
-#' @importFrom utils installed.packages
-#' @import rlang
 #' @import fs
+#' @import rlang
```

Comment:
Would you mind reverting all the changes to this file?

Reply:
Done in b4e10125